docs/agents/tektronix3021c.rst

```diff
@@ -0,0 +1,85 @@
+.. highlight:: rst
+
+.. _tektronix3021c:
+
+===================
+Tektronix AWG Agent
+===================
+
+This agent uses Standard Commands for Programmable Instruments (SCPI)
+It works for many function generators, including the Tektronix3021c.
```

Comment:
If this works for many function generators, not just the Tektronix3021c, should this Agent be more generally named? Like we did with the SCPI PSU Agent? (We can perhaps do this renaming later if it becomes clear we're using more than just the Tektronix device, I know this is time sensitive.)